code/modules/clothing/suits/armor.dm

```diff
@@ -228,7 +228,7 @@
 	item_state = "centcom"
 	w_class = WEIGHT_CLASS_BULKY
 	body_parts_covered = CHEST|GROIN|LEGS|FEET|ARMS|HANDS
-	allowed = list(/obj/item/gun/energy, /obj/item/melee/baton, /obj/item/restraints/handcuffs, /obj/item/tank/internals/emergency_oxygen, /obj/item/tank/internals/plasmaman)
+	allowed = list(/obj/item/gun/energy, /obj/item/melee/baton, /obj/item/restraints/handcuffs, /obj/item/tank/internals/emergency_oxygen, /obj/item/, /obj/item/tank/internals/ipc_coolant)
```

Comment:
gotta make sure plameme still get to use it

Reply:
also, this would mean that literally any item could be stored

Reply:
oh no that's a very bad typo